app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    person_list = []
+
+    for peoples in people:
+        person_list.append(Person(peoples["name"], peoples["age"]))
```

Comment:
You can also simplify this construction a bit by using list comprehension.

Reply:
It's not a good idea to add its type to the variable name; it's enough to just name it in plural.